coverage/lcov-report/base.css

```
@@ -0,0 +1,362 @@
+    border-right: none;
+}
+.coverage-summary th {
+    text-align: left;
+    font-weight: normal;
+    white-space: nowrap;
+}
+.coverage-summary th.file {
+    border-right: none !important;
+}
+.coverage-summary th.pct {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)